### PR TITLE
Improve logging - satosa.frontends.saml2

### PR DESCRIPTION
**Pull request by sebulibah**
### All Submissions:

* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
* [ ] Have you added an explanation of what problem you are trying to solve with this PR?
* [ ] Have you added information on what your changes do and why you chose this as your solution?
* [ ] Have you written new tests for your changes?
* [ ] Does your submission pass tests?
* [ ] This project follows PEP8 style guide. Have you run your code against the 'flake8' linter?

### CODE REVIEW

File: src/satosa/frontends/saml2.py [hunk 1]
```diff
@@ -343,8 +348,9 @@ def _handle_authn_response(self, context, internal_response, idp):
             name_qualifier=None,
         )
 
-        dbgmsg = "returning attributes %s" % json.dumps(ava)
-        satosa_logging(logger, logging.DEBUG, dbgmsg, context.state)
+        msg = "returning attributes {}".format(json.dumps(ava))
+        logline = lu.LOG_FMT.format(id=get_session_id(context.state), message=msg)
```
Comment (ioparaskev) [inline src/satosa/frontends/saml2.py]: you've probably seen this from the failed tests but for the sake of obviousness this needs to be written as `lu.get_session_id`

Comment (sebulibah) [reply src/satosa/frontends/saml2.py]: Thanks.

File: src/satosa/frontends/saml2.py [hunk 2]
```diff
@@ -376,22 +382,26 @@ def _handle_authn_response(self, context, internal_response, idp):
         try:
             args['sign_alg'] = getattr(xmldsig, sign_alg)
         except AttributeError as e:
-            errmsg = "Unsupported sign algorithm %s" % sign_alg
-            satosa_logging(logger, logging.ERROR, errmsg, context.state)
+            msg = "Unsupported sign algorithm {}".format(sign_alg)
+            logline = lu.LOG_FMT.format(id=lu.get_session_id(context.state), message=msg)
+            logger.error(logline)
             raise Exception(errmsg) from e
```
Comment (c00kiemon5ter) [inline src/satosa/frontends/saml2.py]: This will break as `errmsg` is now replaced by `msg`.

File: src/satosa/frontends/saml2.py [hunk 3]
```diff
@@ -376,22 +382,26 @@ def _handle_authn_response(self, context, internal_response, idp):
...
 
         try:
             args['digest_alg'] = getattr(xmldsig, digest_alg)
         except AttributeError as e:
-            errmsg = "Unsupported digest algorithm %s" % digest_alg
-            satosa_logging(logger, logging.ERROR, errmsg, context.state)
+            msg = "Unsupported digest algorithm {}".format(digest_alg)
+            logline = lu.LOG_FMT.format(id=lu.get_session_id(context.state), message=msg)
+            logger.error(logline)
             raise Exception(errmsg) from e
```
Comment (c00kiemon5ter) [inline src/satosa/frontends/saml2.py]: same as above.

**Pull request merged**